README.md

```diff
@@ -74,13 +74,15 @@ on: [ push ]
 
 jobs:
   gitops:
-    uses: Staffbase/gha-workflows/.github/workflows/template_gitops.yml@v1.10.0
+    uses: Staffbase/gha-workflows/.github/workflows/template_gitops.yml@v1.11.0
```

Comment:
Could we create the release after the PR from Kai is also merged? 🤔

Reply:
Sure, if doesn't take days... 😅